src/libtexture/imagecache_pvt.h

```diff
@@ -278,8 +287,8 @@ class OIIO_API ImageCacheFile final : public RefCnt {
         bool full_pixel_range    = false;  ///< data window matches image window
         bool is_constant_image   = false;  ///< Is the image a constant color?
         bool has_average_color   = false;  ///< We have an average color
-        std::vector<float> average_color;  ///< Average color
         spin_mutex average_color_mutex;    ///< protect average_color
+        std::vector<float> average_color;  ///< Average color
```

Comment:
Aside: I'd like to find or create a good `small_vector<T, size>` implementation -- a template that acts like a vector, but can store up to size elements inside itself and only dynamically allocates if more than this expected/typical amount. This would be an ideal place to use a `small_vector<float,4>` where for the usual case of 1-4 channels no heap would be needed.